src/props.fonts.js

```diff
@@ -1,25 +1,55 @@
+  "--font-sans":
+    "system-ui,-apple-system,Segoe UI,Roboto,Ubuntu,Cantarell,Noto Sans,sans-serif",
+  "--font-serif": "ui-serif,serif",
+  "--font-mono":
+    "Dank Mono,Operator Mono,Inconsolata,Fira Mono,ui-monospace,SF Mono,Monaco,Droid Sans Mono,Source Code Pro,monospace",
+  "--font-system-ui": "system-ui, sans-serif",
+  "--font-transitional":
+    "Charter, 'Bitstream Charter', 'Sitka Text', Cambria, serif",
+  "--font-old-style":
+    "'Iowan Old Style', 'Palatino Linotype', 'URW Palladio L', P052, serif",
+  "--font-humanist":
+    "Seravek, 'Gill Sans Nova', Ubuntu, Calibri, 'DejaVu Sans', source-sans-pro, sans-serif",
+  "--font-geometric-humanist":
+    "Avenir, Montserrat, Corbel, 'URW Gothic', source-sans-pro, sans-serif",
+  "--font-classical-humanist":
+    "Optima, Candara, 'Noto Sans', source-sans-pro, sans-serif",
+  "--font-neo-grotesque":
+    "Inter, Roboto, 'Helvetica Neue', 'Arial Nova', 'Nimbus Sans', Arial, sans-serif",
+  "--font-monospace-slab-serif": "'Nimbus Mono PS', 'Courier New', monospace",
+  "--font-monospace-code":
+    "ui-monospace, 'Cascadia Code', 'Source Code Pro', Menlo, Consolas, 'DejaVu Sans Mono', monospace",
+  "--font-industrial":
+    "Bahnschrift, 'DIN Alternate', 'Franklin Gothic Medium', 'Nimbus Sans Narrow', sans-serif-condensed, sans-serif",
+  "--font-rounded-sans":
+    "ui-rounded, 'Hiragino Maru Gothic ProN', Quicksand, Comfortaa, Manjari, 'Arial Rounded MT', 'Arial Rounded MT Bold', Calibri, source-sans-pro, sans-serif",
+  "--font-slab-serif":
+    "Rockwell, 'Rockwell Nova', 'Roboto Slab', 'DejaVu Serif', 'Sitka Small', serif",
+  "--font-antique":
+    "Superclarendon, 'Bookman Old Style', 'URW Bookman', 'URW Bookman L', 'Georgia Pro', Georgia, serif",
+  "--font-didone":
+    "Didot, 'Bodoni MT', 'Noto Serif Display', 'URW Palladio L', P052, Sylfaen, serif",
+  "--font-handwritten":
+    "'Segoe Print', 'Bradley Hand', Chilanka, TSCu_Comic, casual, cursive",
 
-  '--font-weight-1': '100',
-  '--font-weight-2': '200',
-  '--font-weight-3': '300',
-  '--font-weight-4': '400',
-  '--font-weight-5': '500',
-  '--font-weight-6': '600',
-  '--font-weight-7': '700',
-  '--font-weight-8': '800',
-  '--font-weight-9': '900',
+  "--font-weight-1": "100",
+  "--font-weight-2": "200",
+  "--font-weight-3": "300",
+  "--font-weight-4": "400",
+  "--font-weight-5": "500",
+  "--font-weight-6": "600",
+  "--font-weight-7": "700",
+  "--font-weight-8": "800",
+  "--font-weight-9": "900",
 
-  '--font-lineheight-00': '.95',
-  '--font-lineheight-0': '1.1',
-  '--font-lineheight-1': '1.25',
-  '--font-lineheight-2': '1.375',
-  '--font-lineheight-3': '1.5',
-  '--font-lineheight-4': '1.75',
-  '--font-lineheight-5': '2',
+  "--font-lineheight-00": ".95",
+  "--font-lineheight-0": "1.1",
+  "--font-lineheight-1": "1.25",
+  "--font-lineheight-2": "1.375",
+  "--font-lineheight-3": "1.5",
+  "--font-lineheight-4": "1.75",
+  "--font-lineheight-5": "2",
```

Comment:
looks like a formatter changed all the quotes to it's preference and made it into the PR. can you trim these quote change commits from the PR please? 🙂

Reply:
Sorry I missed that 😅

Reply:
Should be fixed